packages/contracts-bedrock/test/kontrol/README.md

```diff
@@ -0,0 +1,69 @@
+# Kontrol Verification
+
+This folder contains Kontrol symbolic property tests.
+
+## Directory structure
+
+The directory structure is as follows
+
+```tree
+test/kontrol
+├── Counter.sol
+├── Counter.t.sol
```

Comment:
We should be able to delete all the Counter code on this PR right?

Reply:
Indeed, before merging the PR these should go away

Reply:
Update and resolve: we'll be keeping the dummy proof for this PR since the execution time is not low enough yet to execute the relevant proofs. Removing the dummy proof and enabling the execution of the proofs introduced in this PR is left for a future PR.